ui/src/components/VolumeListTable.js

```diff
@@ -163,7 +163,7 @@ function GlobalFilter({
     } else {
       searchParams.set('search', value);
     }
-    history.push(`?${searchParams.toString()}`);
+    history.replace(`?${searchParams.toString()}`);
```

Comment:
Here we should keep the `history.push` too ;)
Otherwise we will lose the history of search